bot/cogs/github.py

```diff
@@ -20,10 +20,10 @@ def __init__(self, bot: Bot) -> None:
 
     @command(aliases=["pullrequest", "pullrequests", "issues"])
     @cooldown(1, 5, type=BucketType.user)
-    async def issue(self, ctx: Context, number: int, repository: str = "HotWired-Bot", user: str = "The-Codin-Hole") -> None:
+    async def issue(self, ctx: Context, issueNum: int, repository: str = "HotWired-Bot", user: str = "The-Codin-Hole") -> None:
```

Comment:
As described before, it's uncommon to use capital letters as word separation in python, please change this to `issue_number` and also update all references to it